app/services/procurer/paperflies.rb

```diff
@@ -0,0 +1,52 @@
+module Procurer
+  class Paperflies < Procurer::Service
+    def initialize
+      url = 'https://5f2be0b4ffc88500167b85a0.mockapi.io/suppliers/paperflies'
+      @response = JSON.parse(RestClient.get(url))
+    end
+
+    def call
+      @response.each do |hash|
+        hotel_data = parse_data(hash)
+
+        destination = Destination.find_or_create_by(id: hotel_data[:destination_id])
+
+        hotel = Hotel.find_or_initialize_by(
+          slug: hotel_data[:id],
+          destination_id: destination.id
+        )
+
+        # assumed that the lat and lng will be the same
+        hotel.lat = hotel_data[:lat] if hotel_data[:lat].present?
+        hotel.lng = hotel_data[:lng] if hotel_data[:lat].present?
+
+        hotel = store_string_attributes(hotel, hotel_data)
+        store_images(hotel, hotel_data)
+        store_amenities(hotel, hotel_data)
```

Comment:
Only paperflies has `store_amenities`. Is this intended? Acme and Patagonia have similar properties too

Reply:
I've updated, thanks for the spot 🙏